vision/standard_object/odlc_text_detection.py

```diff
@@ -0,0 +1,132 @@
+"""
+Method for reading the text on the odlc standard objects.
+"""
+
+from typing import Any, Dict, List
+
+import sys
+import cv2
+import numpy as np
+import numpy.typing as npt
```

Comment:
Noticed this after going through the rest of the file, but you should be using the `nptyping` import instead of `numpy.typing`. The `nptyping` module gives a lot more functionality for typing ndarrays. Take a look at vision > common > standard_object > odlc_contour_filtering.py for some examples of how to use this.

Although, I'm not sure you will need to use this at all if you import the Image type alias.